2333/2/libs/rapper.rb

```diff
@@ -0,0 +1,30 @@
+require 'russian_obscenity'
+
+# class Rapper
+class Rapper
+  attr_reader :name, :battles, :number_of_bad_words
+  ROUNDS_ON_BATTLE = 3
+
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+    @number_of_bad_words = bad_words
+  end
+
+  def bad_words
+    RussianObscenity.dictionary = [:default, './my_dictionary.yml']
+    RussianObscenity.find(@battles.join(' ')).map { |word| battles_words.count(word) }.reduce(:+)
```

Comment:
I think it's not immediately obvious what happens here because the line is pretty long. We can reduce it a bit by extracting `RussianObscenity.find(@battles.join(' '))` to a separate method and naming it properly.